data/en/usecases.yaml

```diff
@@ -1,8 +1,8 @@
 - title: Container Security
-  description: Secure containerized applications, no matter what scale, using the power of eBPF.
+  description: Detect malicious behavior in containerized applications, no matter what scale, using the power of eBPF.
   url: /about/use-cases/#protect-containers-kubernetes-and-cloud-native-applications
 - title: Host Security
-  description: Protect your applications in real time wherever they run, whether bare metal or VMs.
+  description: Oversee your applications' behavior in real time wherever they run, whether bare metal or VMs.
```

Comment:
Maybe just personal preference to consistently use the most commonly used terms, e.g. here `Monitor your ...`